kodilla-spring/src/test/java/com/kodilla/spring/portfolio/BoardTestSuite.java

```diff
@@ -50,51 +53,62 @@ public void testBoardHaveDoneListBean() {
         Assert.assertTrue(actual);
     }
 
-
     @Test
-    public void testAddToDoTask() {
+    public void testBoardHaveBoard() {
         //Given
         ApplicationContext context =
                 new AnnotationConfigApplicationContext(BoardConfig.class, Board.class);
-        Board board = context.getBean(Board.class);
-        TaskList task = board.getToDoList();
-        List<String> theTask = task.getTasks();
-        theTask.add("task to do");
-        String expected = "task to do";
+        context.getBean(Board.class);
         //When
-        String actual = theTask.get(0);
+        boolean actual = context.containsBean("board");
         //Than
-        Assert.assertEquals(expected, actual);
+        Assert.assertTrue(actual);
     }
 
     @Test
-    public void testAddInProgressTask() {
+    public void testTaskListToString() {
         //Given
         ApplicationContext context =
                 new AnnotationConfigApplicationContext(BoardConfig.class, Board.class);
         Board board = context.getBean(Board.class);
-        TaskList task = board.getInProgressList();
-        List<String> theTask = task.getTasks();
-        theTask.add("task in progress");
-        String expected = "task in progress";
+        TaskList tasksDone = board.getDoneList();
+        List<String> theTaskDone = tasksDone.getTasks();
+        theTaskDone.add("done task");
+        String expected = "done task\n";
         //When
-        String actual = theTask.get(0);
+        String actual = tasksDone.toString();
         //Than
         Assert.assertEquals(expected, actual);
     }
 
     @Test
-    public void testAddDoneTask() {
+    public void testTaskAdd() {
         //Given
         ApplicationContext context =
                 new AnnotationConfigApplicationContext(BoardConfig.class, Board.class);
         Board board = context.getBean(Board.class);
-        TaskList task = board.getDoneList();
-        List<String> theTask = task.getTasks();
-        theTask.add("done task");
-        String expected = "done task";
+
+        TaskList tasksTodo = board.getToDoList();
+        List<String> theTaskToDo = tasksTodo.getTasks();
+        theTaskToDo.add("task to do");
+
+        TaskList tasksInProgress = board.getInProgressList();
+        List<String> theTaskInProgress = tasksInProgress.getTasks();
+        theTaskInProgress.add("task in progress");
+
+        TaskList tasksDone = board.getDoneList();
+        List<String> theTaskDone = tasksDone.getTasks();
+        theTaskDone.add("done task");
+
+        String expected = "task to do task in progress done task";
```

Comment:
jakos brzydko to wyglada, brak jakiegos formatowania. sprobuj wywolac board.toString() i sprawdzic, czy spodziewasz sie takiego rezultatu.